src/constants.cpp

```diff
@@ -2165,15 +2208,18 @@ const char *cyber_types[] = {
   "Balance Tail",
   "Body Compartment",
   "CyberFin",
-  "CyberSkull",
-  "CyberTorso",
```

Comment:
Removing these line items actually breaks all cyberware. Remember that the key for this table is in awake.h, and removing things here does not remove it there. The effect of this change is to rename everything to what it is two spaces down-- cyber skulls will now appear as dermal sheathing, torsos as foot anchors, etc. I'll fix it on my side.